uberfire-api/src/main/java/org/uberfire/workbench/category/Others.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.uberfire.workbench.category;
+
+import javax.enterprise.context.ApplicationScoped;
+
+@ApplicationScoped
+public class Others extends Category {
+
+    private static final String OTHERS = "OTHERS";
```

Comment:
What makes me think the first thing we're asked to do is i18n these category names!?

Fortunately you can ```@Inject TranslationService ts``` and use that.

Reply:
@manstis this is not for i18n, I have a place for it. This is just a String for some internal stuffs, but they are not intended to be shown to the user

Reply:
@adrielparedes Thanks for the clarification.